acceso/views.py

```diff
@@ -82,15 +78,10 @@ def caso_index(request):
     return render(request, 'acceso/caso_index.html')
 
 
-class casoTable(Resource):
```

Comment:
We don't really need a full third-party library for doing this, and `django-simple-rest` doesn't support Python 3 anyway so we can't use it.